src/website/views/components/dropdown.pug

```diff
@@ -399,42 +399,46 @@ h3 Radio Buttons
     .a-col.-w--12.-w-sm--7.-w-md--8.-w-lg--6.-w-xl--4
       .m-dropdown__menu.-active
         .m-dropdown__menu-item
-          input(class="a-input", type="radio", name="radios", id="radio1")
-          label(for="radio1")
-          label.-text(for="radio1") Radio Button 1
+          .a-radio
+            input(class="a-radio__input", type="radio", name="radios", id="radio1")
+            label.a-radio__label(for="radio1") Radio Button 1
         .m-dropdown__menu-item
-          input(class="a-input", type="radio", name="radios", id="radio2", disabled)
-          label(for="radio2")
-          label.-text(for="radio2") Disabled Radio Button 2
+          .a-radio
+            input(class="a-radio__input", type="radio", name="radios", id="radio2", disabled)
+            label.a-radio__label(for="radio2") Disabled Radio Button 2
         .m-dropdown__menu-item
-          input(class="a-input", type="radio", name="radios", id="radio3")
-          label(for="radio3")
-          label.-text(for="radio3") Radio Button 3
+          .a-radio
+            input(class="a-radio__input", type="radio", name="radios", id="radio3")
+            label.a-radio__label(for="radio3") Radio Button 3
         .m-dropdown__menu-item
-          input(class="a-input", type="radio", name="radios", id="radio4")
-          label(for="radio4")
-          label.-text(for="radio4") Radio Button 4
+          .a-radio
+            input(class="a-radio__input", type="radio", name="radios", id="radio4")
+            label.a-radio__label(for="radio4") Radio Button 4
   :code(lang="html")
     <div class="m-dropdown__menu">
       <div class="m-dropdown__menu-item">
-        <input class="a-input" type="radio" name="radios" id="radio1">
-        <label for="radio1"></label>
-        <label class="-text" for="radio1">Radio Button 1</label>
+        <div class="a-radio">
+          <input class="a-radio__input" type="radio" name="radios" id="radio1">
+          <label class="a-radio__label" for="radio1">Radio Button 1</label>
+        </div>
       </div>
       <div class="m-dropdown__menu-item">
-        <input class="a-input" type="radio" name="radios" id="radio2" disabled="">
-        <label for="radio2"></label>
-        <label class="-text" for="radio2">Disabled Radio Button 2</label>
+        <div class="a-radio">
+          <input class="a-radio__input" type="radio" name="radios" id="radio2" disabled="">
```

Comment:
```suggestion
          <input class="a-radio__input" type="radio" name="radios" id="radio2" disabled>
```
Can we make it just `disabled` instead of `disabled=''`?